extensions/2.0/Vendor/EXT_manifold/README.md

```diff
@@ -0,0 +1,171 @@
+# EXT\_manifold
```

Comment:
Name change here at the top of the README.  Additionally, please rename this entire folder from `EXT_manifold` to `EXT_mesh_manifold`.